### PR TITLE
Stop storing volume time-stepper errors

### PR DESCRIPTION
**Pull request by wthrowe**
## Proposed changes

Instead, calculate the error norm we use immediately and store that (a double).

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

File: src/Time/StepChoosers/ErrorControl.hpp [hunk 1]
```diff
@@ -311,4 +326,70 @@ struct IsUsingTimeSteppingErrorControlCompute
...
+    if (const auto* const error_control =
+            dynamic_cast<const ::StepChoosers::ErrorControl_detail::
+                             ErrorControlBase<EvolvedVariableTag>*>(
+                &step_chooser);
+        error_control != nullptr) {
+      const auto this_tolerances = error_control->tolerances();
+      if (tolerances->has_value() and tolerances->value() != this_tolerances) {
+        ERROR_NO_TRACE(
+            "All ErrorControl events for the same variables must use the same "
+            "tolerances.");
```
Comment (nilsdeppe) [inline src/Time/StepChoosers/ErrorControl.hpp]: Would it be helpful to print the variables tag name at the end of the error? that way users can at least know what to look at in the input file a bit better.

File: tests/Unit/Time/Tags/Test_StepperErrorTolerances.cpp [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <string>
+
+#include "Helpers/DataStructures/DataBox/TestHelpers.hpp"
+#include "Helpers/DataStructures/TestTags.hpp"
```
Comment (kidder) [inline tests/Unit/Time/Tags/Test_StepperErrorTolerances.cpp]: not needed

**Review by kidder — CHANGES_REQUESTED** [review]
you can squash the change

**Comment by wthrowe** [comment]
Squashed both directly.

**Review by kidder — APPROVED** [review]

**Pull request merged**